cli/cmd/lql_create.go

```diff
@@ -30,9 +28,71 @@ var (
 	queryCreateCmd = &cobra.Command{
 		Use:   "create",
 		Short: "create a query",
-		Long:  `Create a query.`,
-		Args:  cobra.NoArgs,
-		RunE:  createQuery,
+		Long: `Create a query.
+
+There are multiple ways you can create a query:
+
+  * Typing the query into your default editor (via $EDITOR)
+  * From a local file on disk using the flag '--file'
+  * From a URL using the flag '--url'
+
+To launch your default editor and create a new query.
+
+    lacework lql create
```

Comment:
Similar comment here as to whether we want to standardize our communication around `lacework query` over the `lacework lql` alias?

Reply:
changed! 👍🏽